### PR TITLE
Upgrade puppet installer

### PR DESCRIPTION
**Pull request by alon-dotan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,13 +49,25 @@ Create your account at https://app.takipi.com/account.html and get your secret k
...
+
 include takipi
 class {'::takipi':
   secret_key => 'YOUR_SECRET_KEY',
 }
 ```
 
+Takipi with specific Java version:
+distribution, will define the type of environment JRE or JDK
+version, will represent the major version
```
Comment (chook) [inline README.md]: remove version

File: manifests/init.pp [hunk 2]
```diff
@@ -78,7 +80,7 @@
     cwd       => '/opt/takipi/etc',
     command   => "/bin/bash ./takipi-setup-package ${secret_key}",
     logoutput => true,
-    unless    => "grep -q ${secret_key} /opt/takipi/work/secret.key",
+    unless    => "/bin/grep -q ${secret_key} /opt/takipi/work/secret.key",
```
Comment (chook) [inline manifests/init.pp:83]: - [x] grep is always at /bin? never /sbin or something?

File: README.md [hunk 3]
```diff
@@ -38,8 +38,8 @@ Takipi installs a daemon process and a JVM agent library on your servers. These
 
 Requires;
 
-* `puppetlabs-stdlib`
-* `puppetlabs-java`
+* `puppetlabs-stdlib`         (puppet module install puppetlabs-stdlib)
```
Comment (chook) [inline README.md:41]: - [x] please also make sure to update the version and add your changes to the CHANGELOG in the readme

File: manifests/init.pp (hunk 2 above)
Comment (alon-dotan) [reply manifests/init.pp:83]: looks like it, tested on centos_6, centos_7, ubuntu_trusty64, ubuntu_xenial64

**Review by pmoust — APPROVED** [review]

**Pull request merged**